src/components/Announce/blogpost.js

```diff
@@ -0,0 +1,6 @@
+export default {
```

Comment:
hmnnn, do u mean fetch data from api for blog repo, like for ui releases? It'll be cool. But need to add a label too.